.drone.yml

```diff
@@ -7,3 +7,44 @@ pipeline:
     - /var/run/docker.sock:/var/run/docker.sock
     commands:
     - dapper ci
+
+  build-all-binaries:
+    privileged: true
+    image: rancher/dapper:1.11.2
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+    - CROSS=1
+    commands:
+    - dapper build
+    when:
+      branch: master
+      event: tag
+   
+
+  stage-binaries:
+    image: rancher/dapper:1.11.2
+    commands:
+    - cp -r ./bin/* ./package/
```

Comment:
With CROSS=1 make will create the binaries in build/bin

Reply:
In the staging step, this just moves the binary for the Docker image. The `scripts/build` still build bin/* binaries, so I think this part is OK.

Reply:
okay, thanks